Shared/Objects/MediaComponents/VideoCodec.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import VideoToolbox
```

Comment:
Whoops, you are right. This was there because a previous version of this I was using Video/AudioToolBox to check if a decoder existed for each type to automatically find compatibility. Eventually I might want to look at that again but that's a whole other PR. 

I've removed this import on the next commit!